source/PreferencesWindow.h

```diff
@@ -1,63 +1,83 @@
 /*
- * Copyright 2009 Haiku, Inc.
+ * Copyright 2024 Nexus6 <nexus6@disroot.org>
  * All Rights Reserved. Distributed under the terms of the MIT License.
  */
-#ifndef _PREFERENCES_WINDOW_H
-#define _PREFERENCES_WINDOW_H
 
+#pragma once
+
+#include "PreferencesWindow.h"
+#include "Preferences.h"
+#include "TabListView.h"
 
 #include <Box.h>
 #include <Button.h>
 #include <CheckBox.h>
+#include <ColorControl.h>
```

Comment:
Generally adding a class reference in the header is enough.
```suggestion
class BColorControl;
```

it's also the case for other members of the class.